pontoon/base/templatetags/helpers.py

```diff
@@ -263,7 +263,8 @@ def provider_login_url(request, provider_id=settings.AUTHENTICATION_METHOD, **qu
     provider = providers.registry.by_id(provider_id)
 
     auth_params = query.get('auth_params', None)
-    query['scope'] = 'profile:uid profile:email profile:display_name'
+    if provider_id != 'google':
+        query['scope'] = 'profile:uid profile:email profile:display_name'
```

Comment:
I think [we can remove](https://github.com/mozilla/pontoon/blob/9353da04929858811867f60961dbc2765ffc1d40/pontoon/settings/base.py#L814) both of these lines completely. I tested locally and it works.

@jotes Could you confirm that?

Reply:
I remember that some docs with suggestions to make the scope of OAUTH request explicit. However, the Jarek from future sees: https://github.com/pennersr/django-allauth/blob/e84a64cc51cd892ecab507f37b0e873be8d81b5d/allauth/socialaccount/providers/fxa/provider.py#L29 and says this line can be removed.

Reply:
Thanks for confirming! @anandthakker So let's indeed remove this.